### PR TITLE
Locale test for PH translation

### PR DESCRIPTION
**Pull request by janahintal**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Locale test for PH translation

**Related github/jira issue (required)**:

Closes #6150 
Related to #5695
**Steps necessary to review your pull request (required)**:

1. pull this branch
2. build and run the app
3. run `npm run e2e:puppeteer locale`
4. It should pass.

**Included in this Pull Request**:
- [x] An e2e or functional test for the bug or feature.
- [ ] A note to the change log.

### CODE REVIEW

File: test/components/locale/locale.puppeteer-spec.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  beforeAll(async () => {
+    await page.goto(url, { waitUntil: ['domcontentloaded', 'networkidle0'] });
+    await page.setViewport({ width: 1920, height: 1080 });
+  });
+
+  it('should run visual test for locale-ph', async () => {
+    await page.waitForSelector('#tabs-2');
+    const img = await page.screenshot();
+    const config = getConfig('locale-ph');
+    expect(img).toMatchImageSnapshot(config);
```
Comment (tmcconechy) [inline test/components/locale/locale.puppeteer-spec.js:16]: @ericangeles does `toMatchImageSnapshot` need an await?

**Review by tmcconechy — CHANGES_REQUESTED** [review]
Just need some lint fixes. Also not sure if an await is needed or not (@ericangeles added this)

File: test/components/locale/locale.puppeteer-spec.js (hunk 1 above)
Comment (ericangeles) [reply test/components/locale/locale.puppeteer-spec.js:16]: @tmcconechy it doesn't need `await` as far as I know (if you mean in line 16).

Comment (tmcconechy) [reply test/components/locale/locale.puppeteer-spec.js:16]: ok then never mind.. Thanks for confirming

**Review by tjamesallen15 — APPROVED** [review]

**Review by ericangeles — APPROVED** [review]

**Pull request merged**